Zend/zend_compile.c

```diff
@@ -4555,6 +4557,20 @@ void zend_compile_break_continue(zend_ast *ast) /* {{{ */
 				depth, depth == 1 ? "" : "s");
 		}
 	}
+
+	if (ast->kind == ZEND_AST_CONTINUE) {
+		int d, cur = CG(context).current_brk_cont;
+		for (d = depth - 1; d > 0; d--) {
+			cur = CG(context).brk_cont_array[cur].parent;
+			ZEND_ASSERT(cur != -1);
+		}
+
+		if (CG(context).brk_cont_array[cur].is_switch) {
+			zend_error(E_DEPRECATED,
+				"Continues targeting switch are deprecated, use break instead");
```

Comment:
Minor nitpick: it took me a second thought to grasp the error message. Wouldn't is be better expressed as “continue targeting switch is deprecated, use break instead”?

Reply:
Agree, that would be better.